pkg/build/controller/strategy/docker.go

```diff
@@ -93,6 +97,10 @@ func (bs *DockerBuildStrategy) CreateBuildPod(build *buildv1.Build, includeAddit
 						EmptyDir: &v1.EmptyDirVolumeSource{},
 					},
 				},
+				{Name: "buildcachedir", VolumeSource: v1.VolumeSource{
+					HostPath: &v1.HostPathVolumeSource{Path: "/miloslavcache/"},
```

Comment:
this needs to exist on the host, right?  or will k8s create it?

Reply:
cri-o/docker should create it if it doesn't exist on mount time right? or does kube check that?

Reply:
i don't know, easy enough to test (create a pod that mounts a non-existent hostdir)

Reply:
We will need to check if we can create this directory on RHCOS.